src/SAML2/AuthnRequest.php

```diff
@@ -307,15 +305,40 @@ public function buildRequestQuery()
      */
     public function getSignedRequestQuery()
     {
-        $queryParams = [self::PARAMETER_REQUEST => $this->rawRequest];
+        $queryParamPairs = explode('&', $this->rawHttpQuery);
```

Comment:
Are all the changes to this class still needed with the new methodology? I vaguely remember just deprecating these urging users to use the new versions instead?